tests/components/mqtt/test_discovery.py

```diff
@@ -268,6 +268,7 @@
     "CONF_DEPRECATED_VIA_HUB",
     "CONF_VIA_DEVICE",
     # Already short
+    "CONF_COMMANDS",
```

Comment:
it's not already short?

Reply:
I added an abbreviation and removed it from this list.